app/assets/main/react_components/footprint_result/ResultContainer.jsx

```diff
@@ -12,89 +12,74 @@ import ResultTitle from './ResultTitle.jsx';
 import SignUpContainer from './SignUpContainer.jsx';
 import WorldComparisonChart from './WorldComparisonChart.jsx';
 import YourFootprintText from './YourFootprintText.jsx';
+import StaticDataProvider from '../context/Footprint/StaticDataProvider.js';
 
 /**
  * React container for Result page components
  */
+const ResultContainer = ({ footprint, projects, countryAverage, registrationsText, commonText, modelText, lifestyleFootprintsText, plan, slug, currency, lang }) => {
     
-const ResultContainer = ({ footprint, projects, countryAverage, registrationsText, commonText, modelText, lifestyleFootprintsText, lang, plan, currency }) => {
```

Comment:
Bara flyttat in allting innanför <StaticDataProvider> och tagit bort onödiga props